crypto/x509/x509_req.c

```diff
@@ -227,13 +227,21 @@ X509_ATTRIBUTE *X509_REQ_get_attr(const X509_REQ *req, int loc)
 
 X509_ATTRIBUTE *X509_REQ_delete_attr(X509_REQ *req, int loc)
 {
-    return X509at_delete_attr(req->req_info.attributes, loc);
+    X509_ATTRIBUTE *attr = NULL;
+
+    attr = X509at_delete_attr(req->req_info.attributes, loc);
```

Comment:
You did not combine the declaration and assignment as I suggested;
```
X509_ATTRIBUTE *attr = X509at_delete_attr(req->req_info.attributes, loc);

if (attr != NULL)
```
but likely better, so because actually all the functions that we are handling here are missing an input validation check (which may be added in a later PR):
```
    if (req == NULL) {
        ERR_raise(ERR_LIB_X509, ERR_R_PASSED_NULL_PARAMETER);
        return NULL;
    }
```
and for accommodating this the above simplification will not work anymore.

Reply:
Oh, I forgot to combine it. Sorry for the inconvenience. In this PR I will only combine declaration and assignment.

Reply:
Well, I wrote, likely better that your forgot :wink:
so no need to change it.